### PR TITLE
Feature US 08

### PR DESCRIPTION
**Pull request by robinschubertdev**

### CODE REVIEW

File: components/FilterBar.js [hunk 1]
```diff
@@ -1,44 +1,68 @@
...
-  onChange,
-  groupedCityEvents,
-  groupedCategoryEvents,
-  filters,
-  reset,
-}) {
+import styled from "styled-components";
+import Button from "./Button";
+import useSWR from "swr";
+const StyledDiv = styled.div`
```
Comment (moonwave99) [inline components/FilterBar.js:6]: Let's call it `StyledContainer` at least, to give a bit more of indication to the developer ^^

File: components/FilterBar.js [hunk 2]
```diff
@@ -1,44 +1,68 @@
...
   };
 
+  const { data: cities } = useSWR("/api/cities");
+  const { data: categories } = useSWR("/api/categories");
+
   return (
-    <div className="filter-container">
-      <button className="filter-toggle" onClick={toggleFilter}>
+    <StyledDiv>
+      <Button color="purple" onClick={toggleFilter}>
```
Comment (moonwave99) [inline components/FilterBar.js:24]: Generally we prefer to pass a `variant` or `mode` prop to buttons, to better depict their role (e.g. `<Button variant="danger">` or `<Button variant="cancel">`), then we pick the associate color via styled components / CSS.

This means that all buttons that do the same thing...must have the same color!

File: components/SearchBar.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  align-items: center;
+`;
+
+export default function SearchBar() {
+  const router = useRouter();
+  const [inputValue, setInputValue] = useState("");
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    router.push(`/search?searchQuery=${inputValue}`);
```
Comment (moonwave99) [inline components/SearchBar.js:18]: `?query=` or `?q=` is enough, á la Google

File: lib/utils.js [hunk 4]
```diff
@@ -8,6 +8,12 @@ export function getURL(filter = {}) {
   filter.city && params.append("city", filter.city);
   return `/api/events${params.size ? `?${params}` : ""}`;
 }
+export function getCategoryURL(filter = {}) {
+  const params = new URLSearchParams();
+  filter.category && params.append("category", filter.category);
+  filter.city && params.append("city", filter.city);
+  return `/api/categories${params.size ? `?${params}` : ""}`;
+}
```
Comment (moonwave99) [inline lib/utils.js:16]: I like this code!

**Review by julialoeschel — APPROVED** [review]
great job! 

one thing for the future: 
Please work on your commit messages. 
- Commit more and comment what you did there. 
- Any AND in a message should be two commits. 
- guide me through your process with these messages 

try it for your next PR
🐜

**Review by julialoeschel — APPROVED** [review]
the better comment would have been: fixed typo in search component. 
please try to work on your comments style.

**Pull request merged**